src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacetValues/DynamicHierarchicalFacetValueRenderer.ts

```diff
@@ -39,23 +38,32 @@ export class DynamicHierarchicalFacetValueRenderer {
     this.button.toggleClass('coveo-with-space', this.shouldHaveMargin);
     this.button.toggleClass('coveo-show-when-collapsed', this.shouldShowWhenCollapsed);
     this.facetValue.isSelected && this.button.setAttribute('disabled', 'true');
-    this.shouldHaveArrow && this.prependArrow();
+    this.shouldHaveBackArrow && this.prependBackArrow();
+    this.shouldHaveForwardArrow && this.appendForwardArrow();
   }
 
-  private prependArrow() {
-    const arrowIcon = $$('div', { className: 'coveo-dynamic-hierarchical-facet-value-arrow' }, SVGIcons.icons.arrowDown);
-    SVGDom.addClassToSVGInContainer(arrowIcon.el, 'coveo-dynamic-hierarchical-facet-value-arrow-svg');
+  private prependBackArrow() {
```

Comment:
Should those arrows be clickable ? 

I think it's reasonable to expect the end user to think they can also click on them to perform the selection.

Reply:
Yes, the whole row is clickable